### ISSUE TITLE
[DISCUSSION] Validating INCAR parameters for external VASP calculations

### ISSUE THREAD

**Issue by matthewkuner** [materialsproject/emmet#772]
Hi all,

I'm currently working on validating the INCAR parameters used in VASP calculations, with the eventual goal of accepting more external data into the MPDB. I was hoping to get your opinions on if/how we should restrict parameters. 

This will be a long post.

Below is a list of INCAR parameters that I believe are in any way relevant. I may have left some out, so please mention any that you think could lead to erroneous results being submitted--we want to account for _every_ possible way to mess up VASP calculations.

Currently, the focus is **only** on ingesting static and relaxation calculations using GGA / GGA+U / R2SCAN.

- [ ] AEXX
- [ ] AMGGAX
- [ ] AMGGAC
- [ ] ALGO / IALGO
- [ ] EDIFF
- [ ] EDIFFG
- [ ] EFIELD
- [ ] ENAUG
- [ ] EPSILON
- [ ] IBRION
- [ ] I_CONTRAINED_M
- [ ] IDIPOL
- [ ] IMAGES
- [ ] ISPIN
- [ ] ISMEAR
- [ ] ISIF
- [ ] IMIX
- [ ] ISYM
- [ ] ISTART
- [ ] IVDW
- [ ] ICHARG
- [ ] LASPH
- [ ] LAMBDA
- [ ] LDAU
- [ ] LDAUTYPE
- [ ] LDAUJ
- [ ] LDAUL
- [ ] LDIPOL
- [ ] LKPROJ
- [ ] LMAXFOCK
- [ ] LMAXMIX
- [ ] LMIXTAU
- [ ] LORBIT
- [ ] LREAL
- [ ] MAGMOM
- [ ] NELM
- [ ] NSW
- [ ] NBANDS
- [ ] NELECT
- [ ] NUPDOWN
- [ ] NWRITE
- [ ] POTIM
- [ ] PSTRESS
- [ ] PREC
- [ ] SYMPREC
- [ ] SIGMA
- [ ] VCA
- [ ] WEIMIN
- [ ]

**Comment by janosh** [comment]
Wow, impressive list! 😄 

I know a bunch of people who might want to weigh in here. Pinging @JaGeo @arosen93 @utf @mkhorton @rkingsbury. If you have time take a look if your favorite INCAR param is already in this list and constrained as you'd expect.

**Comment by arosen93** [comment]
This probably deserves a meeting :) Happy to chat though!

**Comment by JaGeo** [comment]
I think this should be a meeting or we work on a shared md file where we all add.our opinions. Otherwise, Impossible.

**Comment by matthewkuner** [comment]
@JaGeo I agree. We could do both, perhaps? I.e. look into these params on our own time so we are more prepared to discuss this during a meeting.

**Comment by janosh** [comment]
@matthewkuner suggested a Google Doc which would prob make collaboration easiest (even during a meeting). Also great at keeping edit history. @matthewkuner If you already have one, you could link it here and give everyone write access.

**Comment by matthewkuner** [comment]
@janosh I don't have one currently, but will make one and share it here before the end of the day.

**Comment by matthewkuner** [comment]
Hello all. Please request access to this Google Doc so we can more easily discuss this: https://docs.google.com/document/d/1Xunv1TiqHsa0s0KNuC-GA6vh71PGEukPyupe4W2IMb0/edit?usp=sharing

**Comment by arosen93** [comment]
@matthewkuner: Great! Let's do a Google Doc and a meeting, which we can transition to a Markdown file later for longevity's sake.

**Comment by janosh** [comment]
Btw, [Google Docs has a markdown mode](https://support.google.com/docs/answer/12014036) so we can just copy-paste later.

**Comment by arosen93** [comment]
Joke comment: What about the undocumented `LMUSIC` INCAR flag, the VASP flag that somehow (?) plays a little tune when you run the calculation? There are some goodies in the source code... 😉

**Comment by janosh** [comment]
I demand we unequivocally refuse unmelodic calcs!

**Comment by JaGeo** [comment]
It would be good to set a deadline and a meeting right away.

**Comment by janosh** [comment]
I just sent out a Zoom invite for next Thu Jul 13 @ 11 - 12 PT, 20 - 21 German time. Let me know if y'all can make that. Happy to shift around as necessary.

**Comment by JaGeo** [comment]
Thanks for doing so and also for adapting the timing.  I usually start working at 6:30 am and an 8-9 pm meeting is very late for me (and out of our working times).
